src/component/tooltip/TooltipView.js

```diff
@@ -488,7 +488,7 @@ export default echarts.extendComponentView({
         ]);
 
         var tooltipTrigger = tooltipModel.get('trigger');
-        if (tooltipTrigger != null && tooltipTrigger !== 'item') {
+        if (tooltipTrigger === null || tooltipTrigger === 'none') {
```

Comment:
`tooltipTrigger` might be `undefined`, so it would be better use `== null` to verify.

Reply:
change like this ``` if (!tooltipTrigger || tooltipTrigger === 'none')```